### ISSUE TITLE
New repository for translated curriculum

### ISSUE THREAD

**Issue by anymoto** [railsbridge/docs#458]
The current Spanish curriculum is under the [Codificadas organization](https://github.com/codificadas/docs), but we'd like to move it from there, now that we're going to update the translations more often.

Do you think we can have it under the RailsBridge organization? cc @aromaron @ferperales @Hckmar9

**Comment by lilliealbert** [comment]
Sounds great to me!!

**Comment by rachelmyers** [comment]
Yeah, I think that's great! Since it's a fork of railsbridge/docs, what should we rename it to? `railsbridge/documentación`?? :smile_cat:  

(Prematurely, I know: It looks like both @anymoto and I could do it, since we're on the `Owners` teams in both orgs.)

**Comment by anymoto** [comment]
I'd go for railsbridge/docs-es :smile:

**Comment by rachelmyers** [comment]
So much better! :smile: 

@lilliealbert and @tjgrathwell, does @anymoto need anyone else's :+1: on your side??

**Comment by alexch** [comment]
On Thu, Apr 9, 2015 at 6:34 PM, Ana Castro <notifications@github.com> wrote:

> I'd go for railsbridge/docs-es [image: :smile:]
>

Excellent. :+1:

**Comment by alexch** [comment]
Although the current docs/ repo has space for an es/ subdir. I started
moving stuff over there but got distracted. Once the docs page code works
with the current "stepfile" base class, it should be good to merge.

On Fri, Apr 10, 2015 at 12:43 PM, Alex Chaffee <alex@stinky.com> wrote:

>
> On Thu, Apr 9, 2015 at 6:34 PM, Ana Castro <notifications@github.com>
> wrote:
>
>> I'd go for railsbridge/docs-es [image: :smile:]
>>
>
> Excellent. :+1:
>
>
>


-- 
Alex Chaffee - alex@stinky.com
http://alexchaffee.com
http://codelikethis.com
http://twitter.com/alexch

**Comment by anymoto** [comment]
Already renamed our repo to docs-es, and I tried to make the transfer but I can't, I guess it's because our repo is a railsbridge/docs fork. @rachelmyers can you help us with that?

**Comment by rachelmyers** [comment]
I can! I think we need to separate it into it's own network and then we should be able to transfer it. I'm on it.

**Comment by FerPerales** [comment]
Any update regarding this? :dancers: cc @anymoto @rachelmyers

**Comment by lilliealbert** [comment]
we were talking yesterday and @tjgrathwell also reminded me of the thing that @alexch said about moving the curriculum into the /es directory within the main docs repo. I think it'd be cool to have everything live in one repo. The main challenge is the git stuff, and it seems like there are some fancy things we could potentially do, but also maybe we don't even need to be fancy? 

The question: How important is keeping the git history of the translations? If not, we could just squash the Spanish curriculum diff and apply it in one megacommit within the /es directory of the main docs repo. This seems much simpler than the other options to me, but definitely loses history.

**Comment by anymoto** [comment]
I don't mind losing history in this case...what do you think @aromaron @FerPerales?

**Comment by FerPerales** [comment]
Neither do I. We can have the original repo as a back up in case we need that history

**Comment by tjgrathwell** [comment]
I'm actually not sure whether it's better to merge the two repos. Moving `docs-es` into the `railsbridge` org seems like a sane first step, but if we merge all the `/sites/es` content into the main `docs`, we need to consider:

* Should all the contributors to `docs-es` have write access to `docs`? Or should that be limited to a few, with additional changes being made by pull request?
* If the `en` and `es` content lives in the same repo, we **could** switch http://es.railsbridge.org/ to point at the main docs repo. In that case, should (some?) contributors to `docs-es` be able to deploy to the main docs app? (This probably doesn't matter much because there's no harm in keeping a separate deployment for `es`, unless there was some paid heroku service we wanted to share between the instances.)
* If we merge the content from the two repos, what would happen to something like [CONTRIBUTING.md](https://github.com/codificadas/docs-es/blob/master/CONTRIBUTING.md), which is translated but lives outside of /sites/?

I don't know how many people are working on `docs-es` and how many plan on working on it in the future, so I'm not really sure what the ideal workflow is here.

**Comment by alexch** [comment]
Losing git history is fine with me.

If they're in a common repo, we can maybe more easily reference
original-language sources, to better track needed re-translations when the
original paragraph changes. Maybe even in an automated way as part of the
test suite.

> Should all the contributors to docs-es have write access to docs? Or
should that be limited to a few, with additional changes being made by pull
request?

I think this can be decided on a case by case basis. Let's ask @anymoto --
which `es` contributors should get write access to the repo? All of them?
Just you?

> we *could* switch http://es.railsbridge.org/ to point at the main docs
repo

Probably better to keep them separate for now. The heroku apps
can/should/may already have a switch to tell them which locale to launch in.

CONTRIBUTING.md
<https://github.com/codificadas/docs-es/blob/master/CONTRIBUTING.md> can be
renamed CONTRIBUTING-en.md, CONTRIBUTING-es.md, etc.









On Sun, Apr 26, 2015 at 11:59 PM, Travis Grathwell <notifications@github.com
> wrote:

> I'm actually not sure whether it's better to merge the two repos. Moving
> docs-es into the railsbridge org seems like a sane first step, but if we
> merge all the /sites/es content into the main docs, we need to consider:
>
>    - Should all the contributors to docs-es have write access to docs? Or
>    should that be limited to a few, with additional changes being made by pull
>    request?
>    - If the en and es content lives in the same repo, we *could* switch
>    http://es.railsbridge.org/ to point at the main docs repo. In that
>    case, should (some?) contributors to docs-es be able to deploy to the
>    main docs app? (This probably doesn't matter much because there's no harm
>    in keeping a separate deployment for es, unless there was some paid
>    heroku service we wanted to share between the instances.)
>    - If we merge the content from the two repos, what would happen to
>    something like CONTRIBUTING.md
>    <https://github.com/codificadas/docs-es/blob/master/CONTRIBUTING.md>,
>    which is translated but lives outside of /sites/?
>
> I don't know how many people are working on docs-es and how many plan on
> working on it in the future, so I'm not really sure what the ideal workflow
> is here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/railsbridge/docs/issues/458#issuecomment-96492442>.
>



-- 
Alex Chaffee - alex@stinky.com
http://alexchaffee.com
http://twitter.com/alexch

**Comment by anymoto** [comment]
We definitely need a way to track needed re-translations if something in the original content changes. But it's not super important to have it right now. 

The `es` contributors team: @hckmar9, @FerPerales, @aromaron and I. We reduced the group so we can handle the work in a better way. 

We agree that the sane first step is moving the `docs-es` repo to the Railsbridge org, so we can start to work on re-translations as soon as posible. We're hosting a workshop within three weeks and we'd like to use the `es` content. :smiley_cat:

**Comment by rachelmyers** [comment]
I just wanted to check in on this. It seems like we all are on board with moving the `docs-es` repo into the Railsbridge org for now. 

Should we push ahead with that and continue the conversation about potentially pulling it into `docs`? Also, should we make a team for the translators? Maybe called `es-contributors`??

**Comment by anymoto** [comment]
Sounds good :+1:
